Tasks/NuGetToolInstaller/nugettoolinstaller.ts

```diff
@@ -21,4 +22,78 @@ async function run() {
     }
 }
 
-run();
\ No newline at end of file
+function logNugetToolInstallerStartupVariables() {
+    try {
+        let agentVersion = taskLib.getVariable('Agent.Version');
+        if (semver.gte(agentVersion, '2.120.0')) {
+            let externalendpoint = null;
```

Comment:
All of these endpoints logic is not relevant here.

Reply:
Fixed.